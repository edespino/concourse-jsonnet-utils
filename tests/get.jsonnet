local concourse = import 'concourse.libsonnet';

{
  get: concourse.Get('git', dependencies = 'prepare'),
}
