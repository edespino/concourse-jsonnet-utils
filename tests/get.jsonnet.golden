{
   "get": {
      "get": "git",
      "passed": [
         "prepare"
      ],
      "trigger": true
   }
}
